marker_ids: [10, 12, 11, 13]
min_marker_detection: 5
rate_filt_velocity: 20
rate_filt_time: 0.1
cam2imuTf: "/parameters/cam2imu.yaml"
camera_frame: "cam3"
soft_data_vector_length: 50
marker_transform_sample_num: 15
use_soft: true